packages/interop/src/bin.ts

```diff
@@ -1,5 +1,4 @@
 #! /usr/bin/env node
-/* eslint-disable no-console */
```

Comment:
no console.log used in this file. found while searching for my own rule disabling comments.